### ISSUE TITLE
License is too restrictive

### ISSUE THREAD

**Issue by thomasbcox** [eostoolkit/eostoolkit#23]
The license for this software is so restrictive as to make it unusable.

**Comment by generEOS** [comment]
Hi Thomas, thanks for raising this issue. We have talked personally with you and other concerned parties, but we would also like to keep an open dialogue.

In a space where brand, image, community, development are all key drivers to being an attractive DPOS based Block Producing candidate we had some concerns that other's may attempt to leverage our effort, and those of our community members, for their own gain. This has been a massive internal conflict for us as a Block Producer where giving back to the community is part of our core platform.

We understand that code constantly changes and that people may make improvements and additions over time... this is the point of open source. However the possibility (or past reality) of other parties entirely rebranding a site and running a "we did this first" campaign using someone else's software leaves a bad taste in our mouths.

We want to do what is best for this community and are willing to make the changes the community desires. This is already a community project and we only want to see it grow. We encourage the entire community to provide their opinion or advice on this topic. 

Thanks 
~ Nathan, Team GenerEOS

**Comment by RalfWeinand** [comment]
Thank you @thomasbcox for starting the discussion about this. We have discussed the license question within the team and - despite the bad experience we have had after sharing the first version of our toolkit as open-source -we have now decided to put eostoolkit.io under the open-source MIT license again. We hope that this project will become the base of many future EOS projects and will help benefit the whole EOS community. Team GenerEOS

https://github.com/eostoolkit/eostoolkit/commit/feec4848e266d193928edae46a3e01b75f2a7b96
